src/storage/persistence/AccountSecureStorage.js

```diff
@@ -28,8 +29,38 @@ export class AccountSecureStorage extends BaseSecureStorage {
     }
 
     /**
-     * Get all the accounts
-     * @returns {Promise<AccountModel[]>}
+     * Updates account
+     * @param newAccount
+     * @returns {Promise<*>}
+     */
+    static async updateAccount(newAccount: AccountModel): Promise<any> {
+        const allAccounts = await this.getAllAccounts();
+        const edited = allAccounts.map(account => {
+            if (account.id === newAccount.id) {
+                return newAccount;
+            } else {
+                return account;
+            }
+        });
+        return this.saveAccounts(edited);
```

Comment:
If account not found, we save random one?

Reply:
No, result remains the same